ships/vi_dar.json

```diff
@@ -0,0 +1,3559 @@
+{
+  "description": "Created in shadow and filled with its creators\u2019 lust for vengeance, the Vi\u2019dar was to be the ultimate weapon of Romulus. Desperate for some defense against the Borg threat, the Romulans fitted their already powerful disruptor weaponry with Borg technology. These tremendous new weapons can tear through even the most durable hull lining, but their real strength lies against a most curious foe: the Borg themselves. The Romulan scientists working on the Vi\u2019dar managed to create a disruptor system with a constantly shifting frequency, making it impossible for Borg ships to adapt their shielding. With a full-scale Borg invasion all but inevitable, the galaxy may very well place their last hopes in this unlikely ally. A ship born of Borg, destined to destroy them.",
+  "blueprints": {
+    "cost": 0,
+    "quantity": 100
+  },
+  "build_costs": {
+    "materials": [
+      {
+        "type": "crystal",
+        "grade": 3,
+        "rarity": "common",
+        "value": 1587
+      },
+      {
+        "type": "crystal",
+        "grade": 3,
+        "rarity": "uncommon",
+        "value": 371
+      },
+      {
+        "type": "crystal",
+        "grade": 3,
+        "rarity": "rare",
+        "value": 42
+      },
+      {
+        "type": "gas",
+        "grade": 3,
+        "rarity": "common",
+        "value": 358
+      },
+      {
+        "type": "gas",
+        "grade": 3,
+        "rarity": "uncommon",
+        "value": 86
+      },
+      {
+        "type": "gas",
+        "grade": 3,
+        "rarity": "rare",
+        "value": 10
+      }
+    ],
+    "others": [
+      {
+        "type": "3\u2605\u00a0Interceptor\u00a0Parts",
+        "value": 2299
+      }
+    ],
+    "resources": [
+      {
+        "type": "tritanium",
+        "value": 13798500
+      },
+      {
+        "type": "dilithium",
+        "value": 689925
+      }
+    ]
+  },
+  "faction": "neutral",
+  "grade": 3,
+  "img_url": "./assets/vi_dar.png",
+  "rarity": "uncommon",
+  "requirements": [
+    {
+      "name": "Shipyard",
+      "level": 25
+    }
+  ],
+  "scrap_info": {
+    "scrapable": false
+  },
+  "ship_ability": {
+    "title": "Cybernetic Augmentation The Vi'dar's Damage is increased by 24000% against Borg hostiles.",
```

Comment:
I think this may need to be split between title and description. The ability is "Cybernetic Augmentation", I think the rest is description

Reply:
There was a line break but maybe not in the vidar file. Might need to manually fix that part